inventories/openstack.yml

```diff
@@ -0,0 +1,5 @@
+plugin: openstack
+fail_on_errors: true
+inventory_hostname: uuid
```

Comment:
we don't _need_ this, and since it's related to compatibility with the script, I might prefer to not have it, because the point of using it this way is to get the "modern" version, as opposed to the legacy version.